### DISCUSSION TITLE
消除特定副本boss技能的汉化

### DISCUSSION THREAD

**Discussion by DragonMyth** [GpointChen/FFXIVChnTextPatch-GP#300] [Q&A]
之前打6.2零式看boss技能原文习惯了， 请问有没有办法通过改csv文件只消除特定副本（P5S-P8S）里boss的技能汉化呢？

**Comment by GpointChen** [comment]
只能透過修改Action.csv中與該boss有關的技能，將翻譯刪除/留白或是該行整行刪除就不會進行替換。

但是必須自行找到哪些是你要修改的技能，遊戲檔案內並沒有額外的線索或metadata可以篩選。通常同一個boss的技能名會放在一起。

需要注意的是，不能使用excel編輯CSV，否則會造成格式錯誤。通常推薦使用一般的文本編輯器進行修改（例如我是用sublime text）。
